src/useMouseWheel.ts

```diff
@@ -0,0 +1,14 @@
+import {useEffect, useState} from 'react'
+
+export default () => {
+    const [mouseWheelScrolled, setMouseWheelScrolled] = useState(0)
```

Comment:
Do you think we could use [`useRafState`](https://github.com/streamich/react-use/blob/master/docs/useRafState.md) here to merge the updates if they come too fast?

Reply:
I think it is not necessary